packages/verification-vue-component/src/components/Verification/items/cards/VerificationCard.vue

```diff
@@ -158,15 +158,18 @@ export default {
         verificationItem: {
             type: Object,
             required: true
+        },
+        net: {
+            type: String,
+            required: true
         }
     },
     computed: {
-        net() {
-            if (this.$parent.$parent.certifactionEthVerifier.certifactionEthClient.eth.currentProvider.host.indexOf('ropsten') >= 0) {
-                return 'ropsten.etherscan.io'
-            } else {
-                return 'etherscan.io'
-            }
+        registerEvents() {
+            return this.verificationItem.events.filter(event => ['register', 'certify'].includes(event.scope) > 0)
+        },
+        revokeEvents() {
+            return this.verificationItem.events.filter(event => event.scope === 'revoke')
         },
         verificationItemType() {
             if (this.verificationItem.error) {
```

Comment:
Have you checked with Stephan and Fabian if they want to show the technical error again? Because I've removed that in the current version.

Reply:
There's a technical error state in the redesign: https://www.figma.com/file/BIBU6a4krHnu0rb5wueX7v/Verification-Tool-DE?node-id=480%3A0

Reply:
Yes, but I had to remove it, because it would be displayed when infura is not available and we still get a result from the offchain verifier. Maybe it's ok now in the redesign.

Reply:
@stephalba What do you think?

Reply:
@alnicoli As long as we have an off-chain result, we shouldn't show a technical error. It should only be shown if no result is available (on- or off-chain).

Reply:
If we don't have a result from both, we would display "file unknown". Maybe it would be possible to check if no result and an error is present, then display the technical error.

Reply:
Changed as discussed in call. I hope it's correct now